link.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from scrapy.linkextractors import LinkExtractor
+from scrapy.http import HtmlResponse, TextResponse, Response
+from six.moves.urllib.parse import urlparse
+import glob
+from timeit import default_timer as timer
+import click
+import io
+
+
+def main():
+    start = timer()
+
+    url = 'http://scrapinghub.com/'
+    link_extractor = LinkExtractor()
+    total = 0
+    for files in glob.glob('sites/index*'):
+
+        f = (io.open(files, "r", encoding="ISO-8859-1"))
```

Comment:
I think it would be better to keep files in utf-8, I think it's a more common encoding these days.